packages/polaris-typeorm/src/handlers/data-version-handler.ts

```diff
@@ -1,72 +1,60 @@
 import { PolarisExtensions, PolarisGraphQLContext } from '@enigmatis/polaris-common';
 import { EntityMetadata } from 'typeorm';
 import { RelationMetadata } from 'typeorm/metadata/RelationMetadata';
-import { DataVersion, PolarisConnection, QueryRunner, SelectQueryBuilder } from '..';
+import { DataVersion, PolarisConnection, PolarisEntityManager, SelectQueryBuilder } from '..';
 import { isDescendentOfCommonModel } from '../utils/descendent-of-common-model';
 
 export class DataVersionHandler {
     public async updateDataVersion<Entity>(
-        context: PolarisGraphQLContext,
         connection: PolarisConnection,
-        runner: QueryRunner,
+        manager: PolarisEntityManager,
     ) {
         const extensions: PolarisExtensions =
-            (context && context.returnedExtensions) || ({} as PolarisExtensions);
+            (manager.context && manager.context.returnedExtensions) || ({} as PolarisExtensions);
         connection.logger.log('log', 'Started data version job when inserting/updating entity');
-        const result = await this.getDataVersionForMutation(runner, connection);
+        const result = await this.selectDataVersionForUpdate(manager, connection);
```

Comment:
you are using connection only for the purpose of logging, if so, send the logger object and use it instead.